tests/snapshots.rs

```diff
@@ -531,6 +537,11 @@ fn convert_wgsl() {
             "binding-arrays",
             Targets::WGSL | Targets::HLSL | Targets::METAL | Targets::SPIRV,
         ),
+        ("multiview", Targets::SPIRV | Targets::GLSL | Targets::WGSL),
+        (
+            "multiview_webgl",
+            Targets::SPIRV | Targets::GLSL | Targets::WGSL,
```

Comment:
Running this for spirv and even wgsl seems unnecessary since we only want to observe how it affects webgl and we the other multiview test already tests them

Reply:
Done!